### PR TITLE
Update fonts and onboarding svgs

### PR DESCRIPTION
**Pull request by 6a68**
@johngruen This one's probably for you :-)

- Remove the full page / save visible step from the onboarding page
- Tweak the onboarding SVGs to make the Save button look more realistic
  - Note, the second image is scaled, so the button looks funny nonetheless :-\
- Use the Photon standard font-family incantation everywhere
  - Aside from tweaking font-weights in the onboarding headers, I haven't done much twiddling here. Could use some love; feel free to land tweaks into this branch.

### CODE REVIEW

**Comment by 6a68** [comment]
This might also be a good opportunity to fix the shots page headers / footer links being blue instead of black (#3268).

Also, the SVG logo on the homepage apparently uses Helvetica shapes instead of text, and I wasn't up for creating a replacement. We can file a followup bug, maybe?

**Comment by 6a68** [comment]
Old SVG / new SVG, onboarding image 2:




old / new, image 4:

**Comment by 6a68** [comment]
I think the onboarding image removal is #3270, fixed by this PR.

**Comment by niharikak101** [comment]
Added a fix for #3268

File: static/css/shot-index.scss [hunk 1]
```diff
@@ -105,6 +105,10 @@ h1 {
     color: $dark-default;
   }
 
+  a {
+    color: $dark-default;
```
Comment (6a68) [inline static/css/shot-index.scss]: @niharikak101 Hmm, it seems funny that the style above this, `.default-color-scheme & a`, isn't getting applied.

Looking at the git history for the shot index page view, it looks like the `default-color-scheme` class may have been [accidentally removed](https://github.com/mozilla-services/screenshots/commit/1efbd956d6397be8c44887d8e47c81a57d981e6f#diff-d45944c551eead471a128c9a716e2a79L34).

And, after trying it out locally, it looks like adding the `default-color-scheme` classname to the landing page fixes the footer color issues on that page, too.

Would you mind removing your changes and adding that classname to the outermost divs in the views for the shotindex and homepage? You can just force-push your changes; ask if you have any questions.

**Comment by niharikak101** [comment]
@6a68 Does this look okay?

**Comment by johngruen** [comment]
@niharikak101 @6a68 Talked to Taipei team, they are firmly against switching to Fira here for the time being, but we should extend our font stack to include system fonts.

**Comment by 6a68** [comment]
@johngruen Updated, mind taking another look?

@niharikak101 looks ok to me, @johngruen may have some feedback

**Comment by johngruen** [comment]
Yeah, let me pull this down and check it out

**Comment by johngruen** [comment]
WFM

**Pull request merged**